rholang/src/main/scala/coop/rchain/rholang/interpreter/Substitute.scala

```diff
@@ -25,6 +25,16 @@ object Substitute {
       case Right(par)  => Right(par)
     }
 
+  def maybeSubstitute(term: Eval)(implicit env: Env[Par]): Either[Eval, Par] =
```

Comment:
Is Either being used here in the error/success idiom?

Reply:
No.